app/src/main/java/com/example/nidoqueue/model/ExpBinomial.java

```diff
@@ -15,8 +15,8 @@
     private final String TYPE = "binomial";
     private ArrayList<Trial> trials;
 
-    public ExpBinomial(User owner, String name, String description, Boolean geoLocation) {
-        super(owner, name, description, geoLocation);
+    public ExpBinomial(User owner, String name, String description, Boolean geoLocation, String regionSelected) {
```

Comment:
ExperimentData uses ZonedDateTime so I'll probably have to expand upon the functionality now that all experiments are getting a region.